Lib/test/test_signal.py

```diff
@@ -1347,7 +1348,7 @@ def handler(signum, frame):
         # All ITIMER_REAL signals should have been delivered to the
         # Python handler
         self.assertEqual(len(sigs), N, "Some signals were lost")
-
+    @support.requires_resource('flaky')
```

Comment:
```suggestion

    @support.requires_resource('flaky')
```

Reply:
So as I said here, this test is only flaky on the free-threaded build so we should just skip it as it is and not mark it as flaky. Sorry for the back-and-forth.